src/SprykerSdk/Zed/AopSdk/Business/OpenApi/Builder/OpenApiCodeBuilder.php

```diff
@@ -0,0 +1,732 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\AopSdk\Business\OpenApi\Builder;
+
+use cebe\openapi\Reader;
+use cebe\openapi\spec\OpenApi;
+use cebe\openapi\spec\Operation;
+use cebe\openapi\spec\PathItem;
+use cebe\openapi\spec\Reference;
+use cebe\openapi\spec\Schema;
+use Doctrine\Inflector\Inflector;
+use Generated\Shared\Transfer\MessageTransfer;
+use Generated\Shared\Transfer\OpenApiRequestTransfer;
+use Generated\Shared\Transfer\OpenApiResponseTransfer;
+use SprykerSdk\Zed\AopSdk\AopSdkConfig;
+use Symfony\Component\Process\Process;
+
+class OpenApiCodeBuilder implements OpenApiCodeBuilderInterface
+{
+    /**
+     * @var string
+     */
+    protected const SPRYKER = 'Spryker';
+
+    /**
+     * @var string
+     */
+    protected const CORE = 'core';
+
+    /**
+     * @var \SprykerSdk\Zed\AopSdk\AopSdkConfig
+     */
+    protected AopSdkConfig $config;
+
+    /**
+     * @var \Generated\Shared\Transfer\OpenApiResponseTransfer
+     */
+    protected OpenApiResponseTransfer $openApiResponseTransfer;
+
+    /**
+     * @var \Doctrine\Inflector\Inflector
+     */
+    protected Inflector $inflector;
+
+    /**
+     * @var string
+     */
+    protected string $sprykMode = 'project';
+
+    /**
+     * @var array
+     */
+    protected array $parseProperties = [];
+
+    /**
+     * @param \SprykerSdk\Zed\AopSdk\AopSdkConfig $config
+     * @param \Doctrine\Inflector\Inflector $inflector
+     */
+    public function __construct(AopSdkConfig $config, Inflector $inflector)
+    {
+        $this->config = $config;
+        $this->inflector = $inflector;
+        $this->openApiResponseTransfer = new OpenApiResponseTransfer();
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\OpenApiRequestTransfer $openApiRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\OpenApiResponseTransfer
+     */
+    public function build(OpenApiRequestTransfer $openApiRequestTransfer): OpenApiResponseTransfer
+    {
+        $openApi = $this->load($openApiRequestTransfer->getTargetFileOrFail());
+
+        $organization = $openApiRequestTransfer->getOrganizationOrFail();
+        if ($organization === static::SPRYKER) {
+            $this->sprykMode = static::CORE; //Set sprykMode based on organization
+        }
```

Comment:
Extract into a method.

Reply:
Please also bring your methods in order. Think of a newspaper, you start with the headline, then a sub-headline, and then the text.

Methods should be added after the first method that calls this method.